org.cubocore.CoreTime.appdata.xml

```diff
@@ -43,17 +43,13 @@
     <url type="help">https://gitlab.com/cubocore/coreapps/coretime/-/blob/master/README.md</url>
     <url type="bugtracker">https://gitlab.com/cubocore/coreapps/coretime/-/issues</url>
     <url type="contact">https://app.element.io/#/room/#cubocore:matrix.org</url>
-    
+
     <releases>
         <release version="4.5.0" date="2023-08-27"/>
         <release version="4.4.0" date="2022-08-02"/>
         <release version="4.3.0" date="2021-12-25"/>
     </releases>
-    
-    <provides>
-        <id>org.cubocore.CoreTime</id>
-    </provides>
```

Comment:
I don't know how this creates a circular relationship. Is there any reference regarding this @stxm ?

Reply:
I changed the docs so you can link directly https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/#circular-component-relation